.gitignore

```diff
@@ -1,22 +1,26 @@
+# Editor related ignores
+*.swp
+.ensime*
+.idea*
```

Comment:
I never understood why we add these, every developer should have them in a global git ignore :S